src/app/search/search.service.ts

```diff
@@ -201,17 +201,20 @@ export class SearchService {
   }
 
   // Initialization Functions
-  initializeBucketStubs() {
+  initializeCommonBucketStubs() {
     return new Map([
       ['Entry Type', '_type'],
       ['Language', 'descriptorType'],
       ['Registry', 'registry'],
       ['Private Access', 'private_access'],
       ['Verified', 'tags.verified'],
       ['Author', 'author'],
-      ['Organization', 'namespace'],
+      ['Namespace', 'namespace'],
       ['Labels', 'labels.value.keyword'],
       ['Verified Source', 'tags.verifiedSource'],
+      ['Verified 2', 'workflowVersions.verified'],
```

Comment:
These names seem pretty non-descriptive (namespace, organization, verified 2, verified source 2). What is this a map used for (i.e. this map could use a code comment as well)